functional-tests/README.md

```diff
@@ -34,13 +35,14 @@ Follow the `.env-example` template to create an `.env` file under `/functional-t
 ## Run tests with Gradle
 
 The following commands will launch the tests with the individual browsers.
+They must be run in the functional test directory.
 The minespace-frontend tests are run by replacing CustomJUnitSpecRunner with CustomJUnitPublicSpecRunner
 
 The core frontend tests here will be run with the following commands
 ./gradlew chromeTest -DchromeTest.single=CustomJUnitSpecRunner
-./gradlew chromeHeadlessTest -DchromeHeadlessTest.single=CustomJUnitSpecRunner //Will run in pipeline as well, download tsf test will fail
+./gradlew chromeHeadlessTest -DchromeHeadlessTest.single=CustomJUnitSpecRunner //Can run in pipeline as well, however download tsf test will fail
```

Comment:
I think we can remove the pipeline comment, you can still point out that download doesn't work because of how chromeheadless handles it